skaffold.yaml

```diff
@@ -14,6 +14,15 @@
 #      corsAllowOrigins:
 #        - "https://{{ .UI_URL }}"
 
+.im-prod-tolerations-config: &im-tolerations-config
```

Comment:
Are these prod specific? Asking because of the `prod` in the key, while its not in the anchor and it seems to be used regardless of the classification

Reply:
Thanks, I missed that one. 👍